jiant/tasks/qa.py

```diff
@@ -423,3 +428,160 @@ def get_metrics(self, reset=False):
             self._score_tracker = collections.defaultdict(list)
 
         return {"f1": f1, "em": em, "avg": (f1 + em) / 2}
+
+
+@register_task("qasrl", rel_path="QASRL/")
+class QASRLTask(SpanPredictionTask):
+
+    CSV_PREDS = True
+
+    def __init__(self, path, max_seq_len, name, **kw):
+        """QA-SRL (Question-Answer Driven Semantic Role Labeling)
```

Comment:
Make a PR to the documentation (jiant-site repo) that mentions that this task is supported, and how to download the data.

Reply:
Submitted.